dropwizard-parent-pom/pom.xml

```diff
@@ -31,17 +31,16 @@
       idea is that these dependencies should only ever change versions when we upgrade to a
       newer version of dropwizard.
     -->
-    <dropwizard.api.key.bundle.version>0.8.5</dropwizard.api.key.bundle.version>
     <dropwizard.configurable.assets.bundle.version>0.8.5</dropwizard.configurable.assets.bundle.version>
     <dropwizard.metrics.datadog.ecs.version>0.2.0</dropwizard.metrics.datadog.ecs.version>
     <dropwizard.redirect.bundle.version>0.8.5</dropwizard.redirect.bundle.version>
```

Comment:
Another option is to remove this bundle and move the https://github.com/mainstreethub/msh-dropwizard-auth-bundle.  That bundle supports api keys, charon & notary.  Just needs a bump from 0.9 to 1.0 (i believe no code changes are needed)